vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go

```diff
@@ -264,6 +264,7 @@ func (c *Controller) reconcileHandler(ctx context.Context, obj interface{}) {
 		c.Queue.AddRateLimited(req)
 		ctrlmetrics.ReconcileErrors.WithLabelValues(c.Name).Inc()
 		ctrlmetrics.ReconcileTotal.WithLabelValues(c.Name, "error").Inc()
+		log.Error(err, "error reconciling")
```

Comment:
Is this a local patch? Keep in mind we don't verify vendor right now in a CI job or anything, so need to be careful.